app/forms/linking_case/confirmation_form.rb

```diff
@@ -0,0 +1,16 @@
+module LinkingCase
```

Comment:
To follow the pattern for copy_case and inline with controller naming this module should be called `LinkCase`